lang/Polski.json

```diff
@@ -11,19 +11,19 @@
         "no_campaign": "Brak dostępnych aktywnych kampanii. Oczekiwanie na nową kampanię..."
     },
     "login": {
+        "unexpected_content": "Nieoczekiwany błąd zawartości, zwykle z powodu przekierowania.\nUpewnij się że nie jest wymagane dodatkowe logowanie bądź potwierdzenie dostępu do internetu.",
         "chrome": {
             "startup": "Uruchamianie Chrome...",
             "login_to_complete": "Naciśnij ponownie zaloguj, aby zakończyć proces ręcznego logowania...",
             "no_token": "Nie znaleziono tokena autoryzacyjnego.",
             "closed_window": "Okno przeglądarki Chrome zostało zamknięte przed zakończeniem procesu logowania."
         },
         "error_code": "Kod błędu logowania: {error_code}",
-        "unexpected_content": "Nieoczekiwany błąd zawartości, zwykle z powodu przekierowania.\nUpewnij się że nie jest wymagane dodatkowe logowanie bądź potwierdzenie dostępu do internetu.",
         "incorrect_login_pass": "Nieprawidłowa nazwa użytkownika lub hasło.",
-        "incorrect_email_code": "Nieprawidłowy kod e-mail.",
+        "incorrect_email_code": "Nieprawidłowy kod z e-maila.",
         "incorrect_twofa_code": "Nieprawidłowy kod 2FA.",
         "email_code_required": "Wymagany kod e-mail. Proszę sprawdzić e-mail.",
-        "twofa_code_required": "Wymagany token 2FA. Proszę podać token 2FA."
+        "twofa_code_required": "Wymagany token 2FA."
```

Comment:
These two are inconsistent now. If you're removing the additional sentence from the 2FA version, the email version should have it removed as well.

I'd normally suggest a Github change here, but it doesn't want to work for this case, for some reason. Strange. You'll need to push out a commit for this yourself.

Reply:
I was suggesting a source from `English.json`.

```
    "login": {
        "unexpected_content": "Unexpected content type returned, usually due to being redirected. Do you need to login for internet access?",
        "chrome": {
            "startup": "Opening Chrome...",
            "login_to_complete": "Complete the login procedure manually by pressing the Login button again.",
            "no_token": "No authorization token could be found.",
            "closed_window": "Chrome window was closed before the login procedure could complete."
        },
        "error_code": "Login error code: {error_code}",
        "incorrect_login_pass": "Incorrect username or password.",
        "incorrect_email_code": "Incorrect email code.",
        "incorrect_twofa_code": "Incorrect 2FA code.",
        "email_code_required": "Email code required. Check your email.",
        "twofa_code_required": "2FA token required."
```